package.nls.json

```diff
@@ -101,5 +106,48 @@
+  "walkthroughs.deployProject.altText": "Deploy Project action",
+  "walkthroughs.buildEntireProject.title": "Build Entire Project",
+  "walkthroughs.buildEntireProject.description": "Before building the entire project, right-click on the project and set the build command to be used based on the desired build tool such as elias in ARCAD Builder or makei in ibmi-bob.\n[Set Build Command](command:vscode-ibmi-projectexplorer.projectExplorer.setBuildCommand)\nRunning a build will first deploy the project and then invoke the build command.\n[Run Build](command:vscode-ibmi-projectexplorer.projectExplorer.runBuild)",
+  "walkthroughs.buildEntireProject.altText": "Run Build action",
+  "walkthroughs.compileIndividualFiles.title": "Compile Individual Files",
+  "walkthroughs.compileIndividualFiles.description": "Similar to the build command, the project's compile command must be set first before compiling any files.\n[Set Compile Command](command:vscode-ibmi-projectexplorer.projectExplorer.setCompileCommand)\nTo compile the active text editor in which you are working in, using the Run Compile action located at the top right of the editor or in the VS Code command palette.\nTo compile an entire directory or a specific file, use the right-click Run Compile action on any file or directory in the File Explorer or under the Source heading.",
+  "walkthroughs.compileIndividualFiles.altText": "Run Compile action",
+  "walkthroughs.runCustomActions.title": "Run Custom Actions",
+  "walkthroughs.runCustomActions.description": "An alternative to using the build or compile command is to leverage the use of Code for IBM i workspace actions.\nTo generate an initial actions.json file, right-click on the project and use the Launch Action Setup action. Here you will be prompted to select from a set of pre-defined actions.\n[Launch Action Setup](command:vscode-ibmi-projectexplorer.projectExplorer.launchActionsSetup)\nAfter having setup the actions, you can invoke them using the Run Action action on the project or from the top right of the active text editor.",
+  "walkthroughs.runCustomActions.altText": "Run Action action",
+  "walkthroughs.visualizeJobLogs.title": "Visualize Job Logs",
+  "walkthroughs.visualizeJobLogs.description": "To diagnose why a build failed, use the Job Log view which displays all of the information you would get in an IBM i job log and the spool files after a build.\n[Open Job Log View](command:jobLog.focus)",
```

Comment:
More about the features of the Job Log Viewer can be found here (https://ibm.github.io/vscode-ibmi-projectexplorer/#/pages/jobLog/viewing-job-logs)